rhc-ose-ansible/roles/registry/tasks/auth_layer_nginx.yaml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Install Nexus Repository
+  yum:
+    name: "{{ nexus_repo_url }}"
```

Comment:
May want to reference this as `default_registry_nexus_repo_url` to avoid clashing with any nexus related variables that may come in the future.